tests/functional/spec/textfield_suggestions.spec.js

```diff
@@ -8,3 +9,25 @@ describe("Suggestions", () => {
     expect($$(".js-autosuggest-listbox li").length).to.not.equal(0);
   });
 });
+
+describe("Suggestions", () => {
+  it("Given I open a textfield with a suggestions url that allows multiple suggestions, when I have entered text and picked suggestion from a list, then after typing more text it will show new suggestions", () => {
+    browser.openQuestionnaire("test_textfield_suggestions.json");
+    $(SuggestionsPage.country()).setValue("United States of America");
+    $(SuggestionsPage.submit()).click();
+    $(MultipleSuggestionsPage.multipleCountry()).click();
+    browser.pause(500);
+    browser.keys("Ita");
+    $("#multiple-country-answer-listbox li").waitForExist();
+    $("#multiple-country-answer-listbox__option--0").click();
```

Comment:
I don't think we should need to pause. The only other place we use pause is for the timeout test, where it is necessary. I understand that without this it was failing the functional test in Github actions, but maybe if tried using `waitForExist` on the option?

```suggestion
    browser.keys("Ita");
    $("#multiple-country-answer-listbox__option--0").waitForExist();
    $("#multiple-country-answer-listbox__option--0").click();
```

Reply:
I tried and functional test failed..

Reply:
Sorry, it does look like a workaround is needed as the javascript doesn't seem to work if the box is selected immediately. Ideally we would hook into a DOM change when the Javascript initialises. I was hoping to use `autosuggest-input--initialised ` (https://github.com/ONSdigital/design-system/blob/6ebe5e75c41daf8324cf2be1ebce388567f7e625/src/components/input/autosuggest/autosuggest.ui.js#L118) but that is set before the event listeners are initialised.

For now I think we should continue with the pause. Can we clarify the comment so that it's clear why the pause exists, so instead of "otherwise this sequence is executed almost instantly" we could say "to allow for the autosuggest Javascript to initialise".

Reply:
Added corrected comment to both `browser.pause()` instances. Thanks!